src/test/java/com/epam/tc/hw2/ex2/tests/CheckEpamUserLoginTest.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.tc.hw2.ex2.tests;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tc.hw2.ex1.ConfProperties;
+import com.epam.tc.hw2.ex2.GeneralWebDriver;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class CheckEpamUserLoginTest extends GeneralWebDriver {
+
+    @BeforeClass
+    public void setWebDriver() {
```

Comment:
make a Base test and move @Before/@After methods there

Reply:
Why should I do this? @Before is used in only one class and @After is used only in one class